change/@fluentui-react-accordion-7c04c866-80b9-42c1-9332-1272b109bc22.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "chore: Update react-icons package to 2.0.245",
```

```suggestion
  "comment": "chore: Update react-icons package to ^2.0.245",
```

Ditto on the other change files.